packages/3dom/demo/index.html

```diff
@@ -160,12 +161,13 @@
     });
   }
 
-  activate3DOM(gltf) {
+  async activate3DOM(gltf) {
     const script = document.querySelector('script[type="3DOM"]');
     const scriptText = script.textContent;
 
     const context = new ThreeDOMExecutionContext();
-    const graft = new ModelGraft('../shared-assets/models/Astronaut.glb', gltf);
+    const graft = new ModelGraft('../shared-assets/models/Astronaut.glb',
+        await CorrelatedSceneGraph.from(gltf));
```

Comment:
Why do we have to await here? The CorrelatedSceneGraph looks synchronous.

Reply:
This is vestigial, will remove.